.gitignore

```diff
@@ -4,3 +4,4 @@
 acceptance/tests/dat/
 .vscode
 ffi/examples/read-table/build
+/build
```

Comment:
i'm curious. what this is for ignoring? :)

Reply:
this is the folder my local `cmake` created - wanted to get a bit more into the FFI stuff and experience all that goodness :D.